epictrack-api/migrations/versions/74e58a15d180_add_simple_title_to_work.py

```diff
@@ -0,0 +1,48 @@
+"""Add simple title column
+
+Revision ID: 74e58a15d180
+Revises: 2edbfb8b9c0e
+Create Date: 2024-01-22 20:09:36.781375
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '74e58a15d180'
+down_revision = '2edbfb8b9c0e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('works', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('simple_title', sa.String(length=30), nullable=True))
```

Comment:
Just a thought, is it possible to have the "title" a derived field. If you look at the work model, workType and project are relationship there. Is it possible to derive it when required rather than saving it redundantly. @saravanpa-aot @salabh-aot thoughts?

Reply:
would you prefer that the current solution be blocked until we have this discussion?

Reply:
That is possible. We have implemented something similar for Staff's full name.

Reply:
how are we moving forward?

- Merge this and do the derived field logic in a separate step (easier for me)
- Block this merge and do it here